code/game/gamemodes/vampire/vampire.dm

```diff
@@ -141,15 +148,21 @@
 				text += "<br><font color='red'><B>The [special_role_text] has failed!</B></font>"
 				feedback_add_details("traitor_success","FAIL")
 
-		world << text
-	return 1
+	return text
```

Comment:
well you've got a good eye, or you saw something fishy on the scoreboard at a round end. I'll make a PR to fix that in a moment.